wordpress-mobile/WordPress-iOS
# Stats Revamp: ViewMore -> 3rd Level details

This PR adds functionality to allow a user to navigate to a 3rd level Details screen akin to current Period Stats -> Details

Fixes #18939

To test:
1. Build and run app
2. Navigate to Stats -> Period Stats
3. Tap period stats details and ensure no crashes / regressions
4. Now enable both of the new stats feature flags and build and run again
```
statsNewAppearance
statsNewInsights
```
2. Scroll to Followers Total card and tap upper right disclosure to navigate to details screen
3. Scroll down and click View more under Referrers.  You should be taken to a 3rd level which is the Referrers Details screen (same details screen that Period Stats -> Details navigates to you on current App Store version).  Clicking on a Referrer should open the link in WebView (just like on current version of AppStore)
4. Navigate back to the 2nd level and scroll to bottom and then click View more on the countries map card.  You should be taken to the Countries details screen
5. Navigate all the way back to top level Stats Insights
6. Scroll to Likes Total Card and tap upper right disclosure to navigate to details screen
7. On the Post and Pages tab click View more.  You should be navigated to the Post and Pages details screen.  Tap a row again to drill through further to the Post Stats screen
8. Navigate back to the top level and scroll to the Comments Total card and click the upper right disclosure to navigate to the 2nd level details screen
9. On the Top commenters card click View more which will navigate you to the Commenters 3rd level details screen
10. Tap back to return to the 2nd level and scroll to the bottom to the Posts card.
11. Click View more which should take you to the Posts details screen
Click a Posts and Pages row which should open the link in WebView

## Regression Notes
1. Potential unintended areas of impact
Period Stats details screen

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
